### PR TITLE
avoid using ILB for a single kubernetes master

### PR DESCRIPTION
**Pull request by anhowe**
Due to kubernetes issue kubernetes/kubernetes#41185, this is required in case ILB restarts.  For 3 and 5 master cases, we are using #608 for tracking.

This change is [Reviewable](https://reviewable.io/reviews/azure/acs-engine/607)

### CODE REVIEW

File: parts/kubernetesagentcustomdata.yml [hunk 1]
```diff
@@ -66,7 +66,7 @@ write_files:
     - name: localcluster
       cluster:
         certificate-authority: /etc/kubernetes/certs/ca.crt
-        server: https://{{WrapAsVariable "masterInternalLbIp"}}:443
+        server: https://{{WrapAsVariable "masterIP"}}:443
```
Comment (rjtsdl) [inline parts/kubernetesagentcustomdata.yml]: The way this PR handle this, is to replace everywhere using masterIP. 
To me, it is a little bit confusing, when i saw "masterIP", which actually is the "masterInternalLbIp". 

Wondering, is there a way to make the name more explicit?

**Comment by rjtsdl** [comment]
Can you also refer to the issue 608? 

You can point out, this PR only partially alleviate the issue.

File: parts/kubernetesagentcustomdata.yml (hunk 1 above)
Comment (anhowe) [inline parts/kubernetesagentcustomdata.yml]: renamed to kubernetesAPIServerIP

**Comment by anhowe** [comment]
@acs-bot test this please

**Comment by anhowe** [comment]
updated comment with #608.

**Pull request merged**